bokeh/models/glyphs.py

```diff
@@ -402,7 +406,15 @@ class ImageRGBA(XYGlyph):
 
     # a canonical order for positional args that can be used for any
     # functions derived from this class
-    _args = ('image', 'x', 'y', 'dw', 'dh', 'dilate')
+    _args = ('image', 'x', 'y', 'dw', 'dh', 'global_alpha', 'dilate')
+
+    # a hook to specify any additional kwargs handled by an initializer
+    _extra_kws = {
+        'alpha': (
+            'float',
+            'a convenience parameter to set the global_alpha'
+        )
+    }
```

Comment:
I'm not sure the `_extra_kws` additions are needed. The convenience `alpha` parameter is (or should be) documented as a universally available kw arg to all glyph methods (just up until now it was a no-op on images). IIRC `_extra_kws` is for things like palette that are convenience args specific to one glyph

Reply:
No problem, will remove the _extra_kws. I wasn't sure about them but thought I'd add them for completion.